perfdash/google-gcs-downloader.go

```diff
@@ -182,13 +182,21 @@ func getBuildData(result JobToCategoryData, prefix string, category string, labe
 	return result[prefix][category][label]
 }
 
-type bucketUtil struct {
+// BucketUtil is the interface that fetches data from a storage service
+type BucketUtil interface {
+	getBuildNumbersFromBucket(job string) ([]int, error)
+	listFilesInBuild(job string, buildNumber int, prefix string) ([]string, error)
+	getFileFromBucket(job string, buildNumber int, path string) ([]byte, error)
```

Comment:
Since this interface is public, you'll want the methods to be public (TitleCase) too
```suggestion
	GetBuildNumbersFromBucket(job string) ([]int, error)
	ListFilesInBuild(job string, buildNumber int, prefix string) ([]string, error)
	GetFileFromBucket(job string, buildNumber int, path string) ([]byte, error)
```